src/reaction/sorption_base.hh

```diff
@@ -103,13 +106,34 @@ public:
     FieldFEScalarVec conc_solid_fe;                     ///< Underlaying FieldFE for each substance of conc_solid.
 
     /// Input data set - fields in this set are read from the input file.
-    FieldSet input_data_set_;
+    FieldSet input_field_set_;
 
     /// Fields indended for output, i.e. all input fields plus those representing solution.
     EquationOutput output_fields;
 
+    /// Instances of FieldModel used in assembly methods
+    Field<3, FieldValue<3>::Scalar > scale_aqua;
```

Comment:
Slight optimization by using no_sorbing_surface_cond in calculation of scale sorbed.

Reply:
Done.